project/src/molecules/system_builder.py

```diff
@@ -1,33 +1,65 @@
 """Methods used to create systems from job statepoint."""
 import mbuild as mb
-import signac
-from mbuild import Box
+from methane_ua import MethaneUA
+from pentane_ua import PentaneUA
 
-from project.src import molecules
 
-from .methane_ua import MethaneUA
+def construct_system(sp):
+    """Construct systems according to job statepoint.
 
+    Parameters
+    ----------
+    sp: dict (from job.sp)
+        Dictionary contains information necessary to construct a system.
+        Stored as state of job. The dictionary should resemble:
+        {"molecule": str,
+         "engine": str,
+         "replica": int,
+         "temperature": float (in K),
+         "pressure": float (in kPa),
+         "ensemble": str,
+         "N_liquid": int,
+         "N_vap": int,
+         "box_L_liq": int (nm),
+         "box_L_vap", int (nm),
+         "init_liq_den": float (g/cm3),
+         "init_vap_den": float (g/cm3),
+         "mass": float (g/mol),
+         "forcefield_name": str,
+         "cutoff_style": str,
+         "r_cut": float (in nm)}
 
-def SystemBuilder(job):
-    """Construct systems according to job statepoint."""
-    # Update this dict as new reicpes are made
+    Returns
+    -------
+    system: mb.Compound
+        The constructed mbuild Compound
+    """
+    system = mb.Compound()
+
+    # Update this dict as new recipes are made
     molecule_dict = {
         "methaneUA": MethaneUA,
-        "pentaneUA": None,
+        "pentaneUA": PentaneUA,
         "benzeneUA": None,
         "waterSPC/E": None,
```

Comment:
Water spce already exists in mbuild itself, If you look at the test_spce.py file, I include the way to access that water model in there.

Reply:
gotcha, will add that in now